controller/getArticleId.controller.js

```diff
@@ -0,0 +1,16 @@
+const fetchArticleData = require("../model/fetchAtricle.model")
+
+const getArticleByID = (req, res) => {
+    const { article_id } = req.params
+   
+
+    
+    fetchArticleData(article_id).then((data) => {
+
+        res.status(200).send(data)
+    }).catch((err) => {
+        res.status(404).send({msg:err.passThisMsg})
```

Comment:
To save you doing this for every endpoint, you should be using a custom error handler (prompted by the use of `next()`) here to deal with this error.

See these notes for more details: https://notes.northcoders.com/courses/js-back-end/error-handling#3-testing-and-handling-errors